### PR TITLE
Only animate transition between pages in kirby-router-outlet when running as app

### LINKED ISSUE #3083: [Enhancement] Disable animation when navigating in kirby-router-outlet on desktop
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Some applications want to preserve the stacked-pages behavior that comes with kirby-router-outlet, but without the animation. This is useful for e.g. desktop apps, where the animation can seem out of place.  

## Describe the solution you'd like
Ideally we should do this via some top level configuration, as disabling the animation is most likely on a per-platform basis across all outlets etc.

`IonicModule` has an `navAnimation` property that we can initialize with a non-operational animation when running on desktop.  

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #3083 

## What is the new behavior?
Pages (components) inside `kirby-router-outlet` will only animate when Kirby is running inside Capacitor (e.g. in an app context). This is done by checking for the [ionic platform services 'hybrid' option](https://ionicframework.com/docs/angular/platform#platforms).

The config checks for the hybrid flag because we do not wish to animate pages when running in a browser on devices - in that case, we still want the web-like experience. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts [hunk 1]
```diff
@@ -1,15 +1,25 @@
 import { NgModule } from '@angular/core';
-import { IonicModule } from '@ionic/angular';
+import { AnimationController, IonicModule, isPlatform } from '@ionic/angular';
 
+const getIonicConfig = () => {
+  let config: any = {
```
Comment (jakobe) [inline libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Consider strong typing - the type can be imported from:
`import { IonicConfig } from '@ionic/core';`
```suggestion
  let config: IonicConfig = {
```

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+    mode: 'ios',
+    inputShims: true,
+    scrollAssist: true,
+    scrollPadding: false,
+  };
```
Comment (jakobe) [inline libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Consider a conditional spread instead:
```
const navAnimationConfig: IonicConfig = !isPlatform('hybrid') && {
  navAnimation: () => new AnimationController().create(),
};

const config: IonicConfig = {
  mode: 'ios',
  inputShims: true,
  scrollAssist: true,
  scrollPadding: false,
  ...navAnimationConfig,
};
```

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+
+  if (!isPlatform('hybrid')) {
```
Comment (jakobe) [inline libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Are we sure we only wanna do this on hybrid? Wouldn't it make sense to get an app-like experience when running in the browser on devices? 🤔

**Review by jakobe — COMMENTED** [review]
Looks good! Only a few suggestions on typing and readability.
Also consider updating the issue to reflect the implementation...

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts (hunk 3 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Just talked to the requester of this feature, and they do want to have a more web-like behavior on mobile web. 
Which means they do not want to have page animations when running directly in browsers on all types of devices.

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts (hunk 2 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Mind blown, very nice!

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts (hunk 1 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Done in f07534032

File: libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts (hunk 2 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts]: Done in f07534032

**Review by jakobe — APPROVED** [review]
Awesome-doodeli-day!

**Pull request merged**